tests/test_nanopub.py

```diff
@@ -49,3 +70,20 @@ def test_nanopub_fetch():
         assert (isinstance(np, FairData))
         assert (np.source_uri == np_uri)
         assert (len(np.data) > 0)
+
+def test_nanopub_rdf():
+    """
+    Test that Nanopub.rdf() is creating an rdf graph with the right features (contexts)
+    for a nanopub.
+    """
+
+    assertionrdf = rdflib.Graph()
+    assertionrdf.add((Nanopub.AUTHOR.DrBob, Nanopub.HYCL.claims, rdflib.Literal('This is a test')))
+
+    generated_rdf = Nanopub.rdf(assertionrdf)
+
+    assert(generated_rdf is not None)
+    assert((None, RDF.type, Nanopub.NP.Nanopublication) in generated_rdf)
+    assert((None, Nanopub.NP.hasAssertion, None) in generated_rdf)
```

Comment:
I don't know how the `in` operator works for `Graphs`. Is `None` used as a wildcard?